schemas/v1.0/azure.yaml.json

```diff
@@ -371,6 +370,94 @@
                     "$ref": "#/definitions/hook"
                 }
             }
+        },
+        "docker": {
+            "type": "object",
+            "description": "This is only applicable when `host` is `containerapp` or `aks`",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string",
+                    "title": "The path to the Dockerfile",
+                    "description": "Path to the Dockerfile is relative to your service",
+                    "default": "./Dockerfile"
+                },
+                "context": {
+                    "type": "string",
+                    "title": "The docker build context",
+                    "description": "When specified overrides the default context",
+                    "default": "."
+                },
+                "platform": {
+                    "type": "string",
+                    "title": "The platform target",
+                    "default": "amd64"
+                },
+                "tag": {
+                    "type": "string",
+                    "title": "The tag that will be applied to the built container image.",
+                    "description": "If omitted, a unique tag will be generated based on the format: {appName}/{serviceName}-{environmentName}:azdev-deploy-{unix time (seconds)}. Supports environment variable substitution. For example, to generate unique tags for a given release: myapp/myimage:${DOCKER_IMAGE_TAG}"
```

Comment:
Do we need to use `azdev` here? That's sort of dated at this point, no?

Reply:
This is what we were using before for default tag name of built container images.  We have full control over this.  Just let me know what format you would prefer.

Reply:
Updated from `azdev` to `azd` for now.